megamek/src/megamek/common/net/connections/AbstractConnection.java

```diff
@@ -284,6 +284,8 @@ public void update() {
             while ((np = readNetworkPacket()) != null) {
                 processPacket(np);
             }
+        } catch (java.io.InvalidClassException ex) {
+            LogManager.getLogger().error(getConnectionTypeText(), ex);
```

Comment:
Are we really intending to just eat the exception?

Reply:
@NickAragua I don't recall why I added this; I think IDEA or the auto checks complained about the uncaught exception.
I can have that block close the connection as well, like the other blocks.